modules/codegen/src/main/scala/com/twilio/guardrail/generators/Http4sServerGenerator.scala

```diff
@@ -501,8 +504,11 @@ object Http4sServerGenerator {
           case Nil        => responseInMatch
           case generators => q"for {..${generators :+ enumerator"response <- $responseInMatch"}} yield response"
         }
-        val routeBody       = entityProcessor.fold[Term](responseInMatchInFor)(_.apply(responseInMatchInFor))
-        val fullRoute: Case = p"case req @ $fullRouteWithTracingMatcher => $routeBody"
+        val routeBody = entityProcessor.fold[Term](responseInMatchInFor)(_.apply(responseInMatchInFor))
+        val fullRoute: Case =
+          p"""case req @ $fullRouteWithTracingMatcher => 
+             handlerWrapper($operationId, req, {$routeBody})
```

Comment:
The main reason for wrapping it in { } is that `routeBody` is a block of code, i need to somehow delimit it (unless i'm missing something) ... the other alternative was:

```scala
          p"""case req @ $fullRouteWithTracingMatcher => 
             val routeBody = () = {
               $routeBody
             }
             handlerWrapper($operationId, req, routeBody())
```

But this will cause problems if parameter `routeBody` is in the path and doesn't really add anything.

Let me know if you prefer the version with lambda so that i can change it, but i think the block is better